src/settings/LoanHistory/ExceptionsList.test.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+} from '@testing-library/react';
+import fireEvent from '@testing-library/user-event';
+
+import '../../../test/jest/__mock__';
+
+import ExceptionCard from './ExceptionCard';
+import ExceptionsList from './ExceptionsList';
+
+import css from './ExceptionsList.css';
+
+jest.mock('./ExceptionCard', () => jest.fn(({
+  onRemoveException,
+  exceptionIndex,
+}) => {
+  return (
+    // eslint-disable-next-line
+    <div
+      data-testid={`exeptionCard-${exceptionIndex + 1}`}
+      onClick={() => onRemoveException(exceptionIndex)}
+    />
+  );
+}));
+
+describe('ExceptionsList', () => {
+  const mockedFields = {
+    map: jest.fn((callback) => mockedFields.values.map(callback)),
+    push: jest.fn(),
+    remove: jest.fn(),
+    values: [
+      'firstTestPatch',
+      'secondTestPatch',
+    ],
+  };
+  const mockedResources = {
+    paymentMethods: {
+      records: [
+        {
+          payments: [
+            {
+              nameMethod: 'money',
+            },
+            {
+              nameMethod: 'card',
+            },
+            {
+              nameMethod: 'card',
+            },
+          ],
+        },
+      ],
+    },
+  };
+  const testIds = {
+    header: 'header',
+    exceptions: 'exceptionSection',
+    button: 'addButton',
+  };
+  const labelIds = {
+    header: 'ui-circulation.settings.loanHistory.exceptionSectionHeader',
+    button: 'ui-circulation.settings.loanHistory.addExceptionButton',
+  };
+
+  const getById = (id) => within(screen.getByTestId(id));
```

Comment:
nit: looks like we can add a ticket to jira like improvement to add this function globally for tests because I see that this function is used in many pull requests.

Reply:
yeah, looks so, but i think it's question to @Dmitriy-Litvinenko

Reply:
Could you please add this to tech debt?

Reply:
done